### DISCUSSION TITLE
GUI display options?

### DISCUSSION THREAD

**Discussion by roserez** [immich-app/immich#7463] [Q&A]
New user. Love it so far.

Two GUI questions:

1) Is there a way to suppress displaying the real asset filename in the details section of Info? When I share an album via an anonymous link, I don't want the file name to be visible because some filenames embed private metadata. I'd prefer not to change the filenames because they are useful for mapping back to the master asset.

2) Can the description added in Info, be shown in the thumbnails view?

Thanks!

- Mike

**Comment by alextran1502** [comment]
Hi Mike, thank you

1. This can be a feature request
2. No, this will clutter the timeline view. I want the viewing experience not to get cluttered with more information

**Comment by roserez** [comment]
Thank you Alex! It's an honor to meet you. Immich is a very impressive package.

Regarding both questions that you answered - I was thinking of both as perhaps part of a some customization settings. Something as simple of binary checkbox settings like "Display description with thumbnails", "Display date with thumbnails", "Display filename with Info" etc. (Lightroom does it as "Library View Options".) Or something full-blown where it can be applied to specific users, albums, etc... which would probably amount to much more work.

I've noticed in some posts that there seems to be popular requests for privacy settings, especially related to NSFW stuff. Maybe these display options would fit into whatever model you create for that.

Thanks again!

- Mike

**Reply by alextran1502** [reply]
The pleasure is mine. 

About the propose customization, do you have a mockup of how the description should show on the thumbnail? taking into account that we each thumbnail is shown with different aspect ratio

**Reply by roserez** [reply]
I see your point. I had not yet uploaded enough photos to force a second line of photos on a single date to see how you pack them. With the spacing already existing for the date, I had assumed the description could be squoze in without increasing the spacing much. I'll think about it and send you a mockup  Two quick thoughts - one is to add bit of space whenever a photo on the line has a description and a second is to display the description on mouse-over.
